lib/matplotlib/testing/decorators.py

```diff
@@ -366,6 +366,10 @@ def image_comparison(baseline_images, extensions=None, tol=0,
 
         If *None*, defaults to all supported extensions: png, pdf, and svg.
 
+        When testing a single extension, it can be directly included in the
+        names passed to *baseline_images*.  In that case, *extensions* should
```

Comment:
```suggestion
        names passed to *baseline_images*.  In that case, *extensions* must
```

I would prohibit this and enforce it via a check in the code. I assume that the code would still work as is, but produce `image.png.png` files, which would just not be nice to have accidentially commited to the repo.

Reply:
done (with some reordering of the logic)